shell/browser/api/electron_api_web_contents.h

```diff
@@ -393,6 +393,8 @@ class WebContents : public gin::Wrappable<WebContents>,
     fullscreen_frame_ = rfh;
   }
 
+  std::string GetMediaSourceID(content::WebContents* request_web_contents);
```

Comment:
Can you move this API's declaration and implementation code to below `SetWebRTCIPHandlingPolicy`? This class has hundreds of methods and I want to keep them grouped.

Reply:
Thanks, fixed in 1715980